### PR TITLE
Add button to create Discord channels in Hunt table

### LINKED ISSUE #708: Make the "create voice chat" button more obvious

### PR DESCRIPTION
**Pull request by akirabaruah**
Fixes #708

[Screencast from 2024-01-11 17-01-37.webm](https://github.com/cardinalitypuzzles/cardboard/assets/6751517/d570b52e-f77b-48b2-a2ca-19388734ed81)

### CODE REVIEW

File: hunts/src/LinkCell.js [hunk 1]
```diff
@@ -92,6 +128,19 @@ const GreenSheetsSvg = () => {
   );
 };
 
+const Clickable = ({ children, onClick, style, title }) => {
```
Comment (rgossiaux) [inline hunts/src/LinkCell.js:131]: I think you want to just replace this component with an `a` tag

Comment (akirabaruah) [reply hunts/src/LinkCell.js:131]: Sounds like I should change from `span` to `button` then?

Comment (rgossiaux) [reply hunts/src/LinkCell.js:131]: Personally I would just rip out the component and change `Clickable` to `button`, at that point all it's doing is applying `text-muted`

Comment (akirabaruah) [reply hunts/src/LinkCell.js]: Finally got around to updating this to `button`. Seems to work just as before. Thanks!

**Comment by akirabaruah** [comment]
@rgossiaux I think this PR's awaiting review. Should still be functioning correctly.

**Review by rgossiaux — APPROVED** [review]
cool, looks nice!

**Comment by rawxfish** [comment]
Akira out of the country so gonna merge for him

**Pull request merged**